<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>201</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>219</width>
    <height>201</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>219</width>
    <height>201</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
    <underline>false</underline>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Chess Clock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="min">
      <property name="text">
       <string>5 min</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="sec">
      <property name="text">
       <string>120 sec</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Chess Clock</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="start">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QProgressBar" name="timer1">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%p</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="switch1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Switch (---s)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QProgressBar" name="timer2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string>%p</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="stop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="switch2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Switch (---s)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
